### ISSUE TITLE
Can't click on button "take selfie" on v6.3.0 (macos)

### ISSUE THREAD

**Issue by gallexis** [Annihil/revolut-emergency#9]
# Bug report

### Expected behaviour:
When I click on the button "Take" to take a selfie, it does nothing

### Current behaviour:
when i click on the button to take a selfie, I can go to the next step

### Info:
version: 6.3.0
system: MacOS 10.15.4
![image](https://user-images.githubusercontent.com/1928063/78650836-c062ab80-78bf-11ea-85ae-62d0a593d605.png)

**Comment by gallexis** [comment]
![image](https://user-images.githubusercontent.com/1928063/78651699-fa807d00-78c0-11ea-8e9d-05d74fe5f21c.png)

**Comment by Annihil** [comment]
In order to send a selfie, the app is writing it to the disk before sending it, but it seems it does not have the right to write on disk anymore -_-"

**Comment by gallexis** [comment]
Yes apparently it's a protection on the last macos version, but maybe if you write the image in a "safe" & writeable place it should work? I'll look at it.

**Comment by gallexis** [comment]
How about writing the image into ram? Using something like [memfs](https://github.com/streamich/memfs)

**Comment by Annihil** [comment]
Does it work if you put r-emergency.app in a folder like Downloads?

**Comment by gallexis** [comment]
Nope :/

**Comment by gallexis** [comment]
You don't think it's interesting to use memfs?

**Comment by eliac7** [comment]
The issue is on Windows too if that helps. I was trying to sign in now but it stucks there.

**Comment by diogoterremoto** [comment]
Same here, using macOS Catalina 10.15.5, whenever the app is placed on `/Application` or `~/Downloads` folder.

**Comment by diogoterremoto** [comment]
So the error occurs in:
https://github.com/Annihil/revolut-emergency/blob/db2566d203f7cff8563bbbc529e201d224119634/src/screens/Biometric.tsx#L80

I wonder if this is even needed? The next line just reads the file and returns it.

**Comment by albertoxamin** [comment]
@Annihil this issue is there even if full disk access is given 
![image](https://user-images.githubusercontent.com/6067659/87329213-e2ab3b80-c536-11ea-806e-b22c931aaef7.png)
